yarn.lock

```diff
@@ -15161,13 +15138,13 @@ good-listener@^1.2.2:
   dependencies:
     delegate "^3.1.2"
 
-good-squeeze@2.1.0:
-  version "2.1.0"
-  resolved "https://registry.yarnpkg.com/good-squeeze/-/good-squeeze-2.1.0.tgz#99cee91d26d591698a899c28e9310bcf3c6ae28a"
-  integrity sha1-mc7pHSbVkWmKiZwo6TELzzxq4oo=
+good-squeeze@5.0.0:
+  version "5.0.0"
+  resolved "https://registry.yarnpkg.com/good-squeeze/-/good-squeeze-5.0.0.tgz#57003c22cbe20abd5dfd564b9a0f4d168d266196"
+  integrity sha1-VwA8IsviCr1d/VZLmg9NFo0mYZY=
```

Comment:
It looks like the latest version of `good-squeeze` is `5.1.0`, and that was published over two years ago. It appears this is deprecated in favor of `@hapi/good-squeeze`. 

Did you check and see if we might be able to upgrade to that instead?
Also, since this is a non-dev dependency, did you look and see if the change from 2.x to 5.x might break anything?